pkg/auth/providers/oidc/oidc_provider.go

```diff
@@ -99,15 +100,16 @@ func (o *OpenIDCProvider) LoginUser(ctx context.Context, oauthLoginInfo *v32.OID
 	if err != nil {
 		return userPrincipal, groupPrincipals, "", err
 	}
-
 	userPrincipal = o.userToPrincipal(userInfo, claimInfo)
 	userPrincipal.Me = true
-
-	for _, group := range claimInfo.Groups {
-		groupPrincipal := o.groupToPrincipal(group)
-		groupPrincipal.MemberOf = true
-		groupPrincipals = append(groupPrincipals, groupPrincipal)
-
+	groupPrincipals = o.getGroupsFromClaimInfo(claimInfo)
+	//setting a bool for group search flag
+	if claimInfo.Groups == nil && claimInfo.FullGroupPath == nil {
```

Comment:
This is flag is only affected on enable and edit for the auth provider, but I cannot move it to the testAndApply func because the claim information cannot be returned from the LoginUser func.  the framework is expect LoginUser to return only specific values.

Reply:
Add this comment as a comment in the code for future generations

Reply:
After fact checking myself, I was able to move this. I was thinking of AuthenticateUser, which was using LoginUser. I was able to ignore the claimsInfo being returned from LoginUser in the AuthenticateUser func so that its not an issue :D